packages/auth/src/common/AuthErrorStrings.ts

```diff
@@ -13,4 +13,62 @@ export enum AuthErrorStrings {
 	NO_USER_SESSION = 'Failed to get the session because the user is empty',
 	NETWORK_ERROR = 'Network Error',
 	DEVICE_CONFIG = 'Device tracking has not been configured in this User Pool',
+
+	/**
+   * **The user is attempting to sign-up with an email already configured in the User Pool.**
+   */
+	EMAIL_ALREADY_EXISTS = 'An account with the given email already exists.',
+
+	/**
+   * **The user provides an invalid password required to sign-in for a valid username.**
+	 *
+	 * @see `AuthErrorStrings.USER_DOES_NOT_EXIST` will occur if username does not exist
+   */
+	INCORRECT_USERNAME_OR_PASSWORD = 'Incorrect username or password.',
+
+	/**
+   * **The user session is invalid and are required to sign-in again.**
+	 * 
+	 * Error encountered in following scenarios:
+	 * - User attempts sign-in and does not respond to MFA challenge quick enough via `confirmSignIn`
+   */
+	INVALID_SESSION = 'Invalid session for the user, session is expired.',
+
+	/**
+   * **The user provided an invalid code when responding to MFA challenge.**
+	 * 
+	 * Error encountered in following scenarios:
+	 * - User attempts sign-in and provides incorrect code to `confirmSignIn`
+   */
+	INVALID_MFA_CODE = 'Invalid code received for user',
+
+	/**
+   * **The user provided an invalid code when responding to an email confirmation.**
+	 * 
+	 * Error encountered in following scenarios:
+	 * - User attempts confirm sign-up and provides incorrect code to `confirmSignUp`
+   */
+	INVALID_EMAIL_CONFIRMATION_CODE = 'Invalid code provided, please request a code again.',
+
+	/**
+   * **The user attempts to confirm their email a second time, when it's already confirmed.**
+	 *
+	 * @see `AuthErrorStrings.USER_IS_ALREADY_CONFIRMED`
+   */
+	USER_CANNOT_BE_CONFIRMED_IS_ALREADY_CONFIRMED = 'User cannot be confirmed. Current status is CONFIRMED',
+
+	/**
+   * **Method performing action on user yields no results**
+	 *
+	 * Error encountered in following scenarios:
+	 * - User attempts sign-in and provides non-existent email or phone to `signIn`
+   */
+	USER_DOES_NOT_EXIST = 'User does not exist.',
```

Comment:
Lets look to update the string to the following
```suggestion
	USER_DOES_NOT_EXIST = 'Username and/or password is incorrect.',
```

Reply:
Marking as resolved.

I don't think this should be updated based on this comment: #9167 (comment)

Let me know if you think differently!

Reply:
this could be vulnerable to a [user enumeration attack](https://www.rapid7.com/blog/post/2017/06/15/about-user-enumeration/)

Reply:
This error message leaves Amplify Auth vulnerable to an enumeration attack and I am in agreement with @sammartinez. This error message is leaking information to malicious users who may want to enumerate all possible active emails. 

The error message should be 'idempotent' such that a bad actor would not be able to distinguish between a non-existent account and incorrect username/password combination.

Reply:
```suggestion
	USER_DOES_NOT_EXIST = 'Username and/or password is incorrect.',
```

Reply:
@evcodes i'm still in disagreement, this is the exact error provided by Cognito, and is something that should instead be fixed upstream in Cognito. These errors are simply a mapping of the EXACT error provided by Cognito so that they can be translated.

Reply:
![image](https://user-images.githubusercontent.com/80279872/145093333-11d0c584-a03e-49bb-b141-a7be7c89a028.png)